main.go

```diff
@@ -5,14 +5,16 @@ package main
 import (
 	"flag"
 	"fmt"
+	"os"
+
 	"github.com/genomagic/master"
 	"github.com/genomagic/prepper"
 )
 
 func main() {
 	fileParam := "fastq"
-	fileValues := "raw_sequence.fastq"
-	fileUsage := "the path to the FASTQ file containing raw sequence data for assembly"
+	fileValues := "dummy_sequence.fastq"
```

Comment:
The value `"dummy_sequence.fastq"` is used in two places. I suggest you create a constant; maybe call it `dummyFastQ`, or something similar.